source/apps/configure/config.txt

```diff
@@ -12,21 +12,21 @@ App Configuration
 
 .. toctree::
    
-   App Services App </manage-apps/configure/config/app>
-   Users & Authentication Providers </manage-apps/configure/config/auth>
-   MongoDB Data Sources </manage-apps/configure/config/data_sources>
-   Atlas Device Sync </manage-apps/configure/config/sync>
-   Functions </manage-apps/configure/config/functions>
-   GraphQL </manage-apps/configure/config/graphql>
-   HTTP Endpoints </manage-apps/configure/config/http_endpoints>
-   Static Hosting </manage-apps/configure/config/hosting>
-   Log Forwarders </manage-apps/configure/config/log_forwarders>
-   Triggers </manage-apps/configure/config/triggers>
-   Values </manage-apps/configure/config/values>
-   Environment Values </manage-apps/configure/config/environments>
-   Third-Party Services </manage-apps/configure/config/services>
-   Create Template Configurations with Expansions </manage-apps/configure/config/template-expansions>
-   Legacy Configuration Files </manage-apps/configure/config/legacy>
+   App Services App </apps/configure/config/app>
+   Users & Authentication Providers </apps/configure/config/auth>
+   MongoDB Data Sources </apps/configure/config/data_sources>
+   Atlas Device Sync </apps/configure/config/sync>
+   Functions </apps/configure/config/functions>
+   GraphQL </apps/configure/config/graphql>
+   HTTP Endpoints </apps/configure/config/http_endpoints>
+   Static Hosting </apps/configure/config/hosting>
+   Log Forwarders </apps/configure/config/log_forwarders>
+   Triggers </apps/configure/config/triggers>
+   Values </apps/configure/config/values>
+   Environment Values </apps/configure/config/environments>
+   Third-Party Services </apps/configure/config/services>
+   Create Template Configurations with Expansions </apps/configure/config/template-expansions>
+   Legacy Configuration Files </apps/configure/config/legacy>
```

Comment:
i thought we were moving all this to reference section. why is it moved here?

Reply:
Separate PRs

Reply:
can you just rebase this PR on top of that one?

Reply:
Done now - it wasn't merged in before